docs/source/vpic.rst

```diff
@@ -193,6 +197,38 @@ Contiguously read the first ``TO_READ_NUM_PARTICLES`` elements:
 	'FILE_PATTERN': 'STRIDED'
 	'READ_OPTION': 'STRIDED'
 
+- 4 patterns for 2D read
+
+1. PRL: Refers to the Peripheral data access pattern. Data is read from the periphery of the file, which is a frame of fixed width and height around the dataset.
```

Comment:
"read from the periphery of the file" --> do you mean "read from the periphery of the 2D dataset"